variables.tf

```diff
@@ -118,3 +118,9 @@ variable "ssl_mode" {
   description = "Set up the SSL mode. If SSL is required, this value is TRUSTED_CLIENT_CERTIFICATE_REQUIRED."
   default     = "ALLOW_UNENCRYPTED_AND_ENCRYPTED"
 }
+
+variable "postgres_version" {
```

Comment:
You'll want to run `make terraform/docs` so that the table in the README gets updated with the new variable:
https://github.com/Datatamer/terraform-gcp-tamr-cloud-sql/blob/master/README.md

Reply:
I'm getting a permission denied error when I try running that command. Do I need to set my PAT somewhere within the repo?

Reply:
I ended up adding it manually... Hopefully it achieves the same effect.